src/main/java/frc/team166/robot/subsystems/Lift.java

```diff
@@ -0,0 +1,180 @@
+import frc.team166.chopshoplib.commands.CommandChain;
+import frc.team166.chopshoplib.commands.SubsystemCommand;
+import frc.team166.robot.RobotMap;
+import frc.team166.robot.RobotMap.Encoders;
+import edu.wpi.first.wpilibj.DigitalInput;
+
+public class Lift extends Subsystem {
+    // Defines Limit Switches (Digital imputs)
+    DigitalInput TopLimitSwitchLift = new DigitalInput(RobotMap.DigitalInputs.LIFT_LIMIT_SWITCH_BOTTOM);
+    DigitalInput BottomLimitSwitchLift = new DigitalInput(RobotMap.DigitalInputs.LIFT_LIMIT_SWITCH_TOP);
+    // Defines Encoders 
+    Encoder liftEncoder = new Encoder(RobotMap.Encoders.LIFT_A, RobotMap.Encoders.LIFT_B);
+    // Defines Motors 
+    WPI_TalonSRX m_liftmotor = new WPI_TalonSRX(RobotMap.CAN.liftMotorA);
+    WPI_TalonSRX m_liftmotor2 = new WPI_TalonSRX(RobotMap.CAN.liftMotorB);
+    // Defines the previus motors as one motor 
+    SpeedControllerGroup liftdrive = new SpeedControllerGroup(m_liftmotor, m_liftmotor2);
+    DoubleSolenoid LiftTransmission = new DoubleSolenoid(RobotMap.Solenoids.LIFT_TRANSMISSION_A,
+            RobotMap.Solenoids.LIFT_TRANSMISSION_B);
+
+    //enumerator that will be pulled from for the GoToHeight Command
+    public enum LiftHeight {
+        //will be changed
+        kFloor(0), kSwitch(1), kPortal(2), kIntake(3), kScaleLow(4), kScaleHigh(5), kClimb(6);
+
+        @SuppressWarnings("MemberName")
+        private double value;
+
+        //
+        LiftHeight(double value) {
+            this.value = value;
+        }
+
+        public double get() {
+            return value;
+        }
+    }
+
+    public Lift() {
+        //creates a child for the encoders
+        addChild(liftEncoder);
+        addChild(TopLimitSwitchLift);
+        addChild(BottomLimitSwitchLift);
+        //will be changed
+        liftEncoder.setDistancePerPulse(.1);
+    }
+
+    //boolean to test for the top limit switch
+    private Boolean TopLimitSwitchOn() {
+        if (!TopLimitSwitchLift.get())
+            return true;
+        else
+            return false;
+    }
+
+    //boolean to test for the bottom limit switch
+    private Boolean BottomLimitSwitchOn() {
+        if (!BottomLimitSwitchLift.get())
+            return true;
+        else
+            return false;
+    }
+
+    //sets the motor's speed, can be called later, these are called later
+    public void goUp() {
+        liftdrive.set(0.25);
+    }
+
+    public void goDown() {
+        liftdrive.set(-0.25);
+    }
+
+    //gear changes
+    public void shiftToHighGear() {
+        LiftTransmission.set(Value.kForward);
+    }
+
+    public void shiftToLowGear() {
+        LiftTransmission.set(Value.kReverse);
+    }
+
+    //does not do anything
+    public void initDefaultCommand() {
+        setDefaultCommand(new SubsystemCommand(this) {
+            @Override
+            protected boolean isFinished() {
+                return false;
+            }
+        });
+    }
+
```

Comment:
Because the default command for this subsystem does nothing, you can delete everything inside the brackets of "public void initDefaultCommand() { }"